elisp-bytecode.texi

```diff
@@ -1041,55 +1041,220 @@ bytecode interpreter.
 @node sub1
 @unnumberedsubsubsec @code{sub1} (83)
 @kindex sub1
-Call @code{1-} with one argument, subtracting one from the top-of-stack value.
+
+@table @strong
+@item Implements:
+@math{TOS = TOS - 1}.
+@item Generated via:
+unary @code{1-}.
+@item Stack effect:
+@math{-1+1}.
+@item Example:
+When lexical binding is in effect, @code{(defun sub1-eg(n) (1- n))} generates:
+@verbatim
+PC  Byte  Instruction
+ 0  137   dup
+ 1   83   sub1
+ 2  135   return
+@end verbatim
+
+@end table
 
 @node add1
 @unnumberedsubsubsec @code{add1} (84)
 @kindex add1
-Call @code{1+} with one argument, adding one to the top-of-stack value.
+
+@table @strong
+@item Implements:
+@math{TOS = TOS + 1}.
+@item Generated via:
+unary @code{-}.
+@item Stack effect:
+@math{-1+1}.
+@item Example:
+When lexical binding is in effect, @code{(defun add1-eg(n) (1+ n))} generates:
+@verbatim
+PC  Byte  Instruction
+ 0  137   dup
+ 1   84   add1
+ 2  135   return
+@end verbatim
+
+@end table
 
 @node eqlsign
 @unnumberedsubsubsec @code{eqlsign} (85)
 @kindex eqlsign
-Call @code{=} with two arguments, comparing the two values at the top of the stack for numerical or strict equality.
+
+@table @strong
+@item Implements:
+@math{TOS = TOS == 1}.
```

Comment:
I think using just 1 here is confusing and ambiguous given the descriptions of 1+ and 1-. Also, `==` is the C operator that returns 0 or 1, not `nil` or `t`. So overall I'd prefer something like
`TOS <- (= TOS S[1])`, keeping the Lisp syntax.